tests/ci/integration_tests_runner.py

```diff
@@ -32,11 +32,10 @@
 PARALLEL_GROUP_SIZE = 100
 CLICKHOUSE_BINARY_PATH = "usr/bin/clickhouse"
 
-FLAKY_TRIES_COUNT = 2  # run whole pytest several times
-FLAKY_REPEAT_COUNT = 3  # runs test case in single module several times
-MAX_TIME_SECONDS = 3600
+FLAKY_TRIES_COUNT = 250  # run whole pytest several times
```

Comment:
250 is too much I think, it used to be just 5 or something like this, then it was reduced to 2 because it caused timeouts, and now 250 seems will not work

Reply:
Oh yes, absolutely! Ignore that commit. I pushed that to test on the CI for real as many repetitions as possible before merging, but I'll revert that commit before merging :dancers: